client/config/dev.config.js

```diff
@@ -1,9 +1,4 @@
 module.exports = {
     APP_NAME: 'navi',
     API_SERVER: 'http://localhost:8081',
-    REGISTER_PATH: '/users/register',
```

Comment:
this is great I would like to see this client side config refactored out all together

Reply:
`client/src/js/server-request-utils.js` has `BASE_ENDPOINTS` object that can be used to refactor accordingly. I believe it currently has all the endpoints being used currently and can be easily expanded as needed for those that are missing.

Reply:
Yes I replaced references to the config *_PATH properties with BASE_ENDPOINTS.

Reply:
@seckboy wonderful!